### ISSUE TITLE
Provide a way to specify execution order of the request declaratively

### ISSUE THREAD

**Issue by thesobercoder** [usebruno/bruno#3068]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Currently, we can rearrange all requests within a folder or collection by dragging and dropping, but that order is not preserved in the configuration like `bruno.json` or any other file. If I want to share this collection, I'm not sure if this order will be preserved as the request order is important.

### Mockups or Images of the feature

Not sure where this should be preserved, but the order is critical.

**Comment by junchuLF** [comment]
I also encountered this problem. Our IT runs using CLI in Docker and must be executed sequentially.

**Comment by riesinger** [comment]
Isn't that exactly what the `seq` field in the `meta` section of `.bru` files is used for? If you drag & drop the order of your requests, you should be able to see that number change in your `.bru` files.

**Comment by thesobercoder** [comment]
> Isn't that exactly what the `seq` field in the `meta` section of `.bru` files is used for? If you drag & drop the order of your requests, you should be able to see that number change in your `.bru` files.

I'm not sure if one `.bru` file can contain multiple requests. In my case, I have multiple `.bru` files in a folder as a set, which I want to execute sequentially.

**Comment by riesinger** [comment]
Yes, that's what I meant. Each `.bru` file will contain a `meta` section, which contains a `seq` field with a number. The number indicates the "index" of the file in the collection, which is how they _should_ be executed sequentially (if you don't use `setNextRequest`). If you re-order the files in the UI via Drag'n'Drop, the `seq` field should be updated in the files. Take a peek into the files 🙂
